scripts/iSnobal/output_pngs.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+"""This script generates PNG files from a list of NetCDF files created from daily iSnobal runs and creates a GIF from the PNG files."""
+
+import xarray as xr
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np
+import pathlib
+
+def parse_args():
+    """
+    Parse command line arguments.
+
+    Returns:
+        argparse.Namespace: Parsed arguments.
```

Comment:
Nice docs!

I have been starting to use type hints instead of documenting the with text:
https://docs.python.org/3/library/typing.html

Some IDE's have nice support for that.

Reply:
Thanks :) I took a stab with type hints, will try to keep this in mind moving forward